shell/platform/windows/flutter_window.cc

```diff
@@ -241,6 +242,40 @@ void FlutterWindow::OnScroll(double delta_x,
                                       device_id);
 }
 
+void FlutterWindow::OnSetFocus() {
+  has_focus_ = true;
+  SendLifecycleEvent();
+}
+
+void FlutterWindow::OnKillFocus() {
+  has_focus_ = false;
+  SendLifecycleEvent();
+}
```

Comment:
Is it possible to receive a `WM_SETFOCUS` or `WM_KILLFOCUS` message even after the window isn't visible? Would that scenario result in a pause event? If so, that seems undesirable.